echo.go

```diff
@@ -563,11 +563,11 @@ func (e *Echo) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Middleware
 	h := func(c Context) error {
 		method := r.Method
-		path := r.URL.RawPath
-		if path == "" {
-			path = r.URL.Path
+		rpath := r.URL.RawPath // raw path
```

Comment:
`rawPath` is find here

Reply:
fixed

Reply:
`path` collides with imported package name.. that's why I use `rpath` with a comment